Xamarin.Forms.Platform.UAP/ViewToRendererConverter.cs

```diff
@@ -93,7 +93,7 @@ protected override Windows.Foundation.Size MeasureOverride(Windows.Foundation.Si
 					result = new Windows.Foundation.Size(request.Width, request.Height);
 				}
 
-				_view.Layout(new Rectangle(0, 0, result.Width, result.Height));
+				Layout.LayoutChildIntoBoundingRegion(_view, new Rectangle(0, 0, result.Width, result.Height));
```

Comment:
These two changes allow the view to be sized properly when there are margins on the container. Before this, the text of the TitleView was not centered properly.

Before:

<img width="1920" alt="before" src="https://user-images.githubusercontent.com/7827070/39593677-614a4292-4ebf-11e8-920c-957445c19572.PNG">

After:

<img width="1920" alt="after" src="https://user-images.githubusercontent.com/7827070/39593681-646e222c-4ebf-11e8-8ac9-f362b6c5249c.PNG">